lib/kitchen/templates/composite_page_template.xhtml.erb

```diff
@@ -1,5 +1,5 @@
-<% main_title_tag = @in_composite_chapter ? 'h3' : 'h2' %>
-<div class="os-eoc os-<%=@container_class_type%>-container" data-type="composite-page" data-uuid-key="<%= @uuid_key %>">
+<% main_title_tag = @in_composite_chapter ? 'h3' : @is_eoc ? 'h2' : 'h1' %>
```

Comment:
this is getting complicated -- can `main_title_tag` become a template variable, and the logic here be moved to the class?